torch/testing/_internal/optests/make_fx.py

```diff
@@ -0,0 +1,87 @@
+import torch
+from torch.fx.experimental.proxy_tensor import make_fx
+from torch.testing._internal.common_methods_invocations import wrapper_set_seed
+import torch.utils._pytree as pytree
+
+
+def make_fx_check(
+    func,
+    args,
+    kwargs,
+    tracing_mode,
+    assert_close=torch.testing.assert_close,
+    randomize_data=False,
+):
+    f, *new_args = handle_sizes_for_dynamic_shapes(func, args, kwargs)
+
+    def run(f, *args, **kwargs):
+        return wrapper_set_seed(f, *args, **kwargs)
+
+    traced_f = make_fx(f, tracing_mode=tracing_mode)(*new_args)
+
+    msg = (
+        "op(*args, **kwargs) and make_fx(op)(*args, **kwargs) produced different "
+        "values. This could mean that your abstract impls (meta/FakeTensor impls) "
+        "are incorrect, that your operator is not completely traceable (e.g., "
+        "it relies on some global state), or that there is a bug in make_fx."
```

Comment:
is it worth including the traced graph in the error message for sanity?

Reply:
we could!

Reply:
On second thought, I don't want to, because it means the error message might get long, especially if you have a CompositeImplicitAutograd op that decomposes into something like matmul. It's pretty easy for the user to do the make_fx(op)(*args, kwargs) themselves and see what it traces out as.